client/packages/requisitions/src/ResponseRequisition/DetailView/IndicatorEdit/IndicatorLineEdit.tsx

```diff
@@ -54,8 +54,11 @@ const InputWithLabel = ({ data }: { data: IndicatorColumnNode }) => {
     data.valueType === IndicatorValueTypeNode.Number ? (
       <NumericTextInput
         width={INPUT_WIDTH}
-        value={Number(draft?.value)}
+        value={isNaN(Number(draft?.value)) ? 0 : Number(draft?.value)}
         onChange={v => {
+          if (isNaN(Number(v))) {
+            update({ value: '0' }).then(errorHandler);
+          }
           update({ value: String(v) }).then(errorHandler);
```

Comment:
Completely stylistic but I'd:
```suggestion
          const newValue = isNaN(Number(v)) ? 0 : v;
          update({ value: String(newValue) }).then(errorHandler);
```